src/main/java/microblogevolved/Main.java

```diff
@@ -0,0 +1,56 @@
+package microblogevolved;
+
+import spark.ModelAndView;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Created by darionmoore on 12/17/16.
+ */
+public class Main {
+    static User user;
+    static String password;
+    static ArrayList<String> messages = new ArrayList();
+    static HashMap<String, User> userList = new HashMap<>();
+
+
+    public static void main(String[] args) {
+        Spark.init();
+
+//        Create a GET route for "/" and a POST route for "/create-user" and "/create-message".
+        Spark.get("/", ((request, response) -> {
+            HashMap model = new HashMap();
+            if(userList.containsValue(user)){
+                userList.containsKey(password);
```

Comment:
This will only work if your key is the password which makes no sense why you would do that